### ISSUE TITLE
[SR-14465] Many images in language guide do not have alt text

### ISSUE THREAD

**Issue by swift-ci** [apple/swift-book#13]
|                  |                 |
   |------------------|-----------------|
   |Previous ID       | SR-14465      |
   |Radar             | rdar://problem/76466123         |
   |Original Reporter | amos (JIRA User)      |
   |Type              | Bug    |

<details>
  <summary>Environment</summary>

Any web browser

</details>

   <details>
  <summary>Additional Detail from JIRA</summary>

   |                  |                 |
   |------------------|-----------------|
   |Votes             | 0         |
   |Component/s       |     |
   |Labels            | Bug, Documentation        |
   |Assignee          | None      |
   |Priority          | Medium      |

   

   md5: 636d25b780d6f98d27615d54de6badbd

  </details>

**Issue Description:**

Many images in the language guide on docs.swift.org are missing meaningful alt text. Please fix so people using screen readers can fully experience the docs.

 

For example: in the [strings and characters section](https://docs.swift.org/swift-book/LanguageGuide/StringsAndCharacters.html), the image illustrating handling of indentation in multi-line strings has the image path as its alt text.

**Comment by krilnon** [comment]
Hey @Nitya-Pasrija, yes, these images need descriptions. If you're still interested in working on this; a PR is welcome! I'm interested as well, so feel free to tag me in discussions. (For context, I think one of our early publication systems didn't propagate image descriptions to the rendered book, so these have been missing for a while.)

**Comment by amartini51** [comment]
Started adding alt text on a branch: https://github.com/amartini51/swift-book/tree/alt_text_76466123
